src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -539,6 +539,9 @@ COOP_PINVOKE_HELPER(UInt8 *, RhGetCodeTarget, (UInt8 * pCodeOrg))
 #elif _TARGET_ARM64_
         UNREFERENCED_PARAMETER(unboxingStub);
         PORTABILITY_ASSERT("@TODO: FIXME:ARM64");
+#elif _TARGET_WASM_
```

Comment:
Nit: Put this into `#else` instead of `#error Unsupported architecture`.

Reply:
Done